### PR TITLE
Step 1

### PR DESCRIPTION
**Pull request by gavinnwang**
## Questions

Please include the following when you are writing your PR:   
General things:

1. What is the purpose of this PR?
**is to understand pybind**
3. What changes did you make? Why?
**I wrote unitests and used cpp module in a python file**
5. What bugs did you find while testing?
**none** 

This PR Specific:

1. What challenges did you face while writing the module from scratch?
**none**
3. How did you ensure your unit tests are comprehensive?
**I made sure it covers edge cases** 
5. Did you have enough guidance to complete the task?
**yes**

### CODE REVIEW

File: src/pricingutil.cpp [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    float result = (prevPrice * (0.9 + interest)) * oleoConstant;
+    this->val = result;
+    return result;
+}
+
+float PricingUtil::getVal() {
+    return this->val;
+}
+
+PricingUtil::PricingUtil() {}
```
Comment (minnce) [inline src/pricingutil.cpp:13]: initialize val here please

**Review by minnce — CHANGES_REQUESTED** [review]
Initialize val please - and just make sure to target the right branches so that GH doesn't blow up and bring up conflicts

File: src/pricingutil.cpp (hunk 1 above)
Comment (gavinnwang) [reply src/pricingutil.cpp:13]: Hey Chris I just fixed this under a new PR called step-1-gavin
